CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
```

Comment:
This probably should start as `##` (the other sub topic headers are `##` too. only `ActiveWaiter` is `#`.)